src/applications/vaos/sass/vaos.scss

```diff
@@ -45,4 +45,11 @@
   .form-radio-buttons label {
     max-width: none;
   }
+}
+
+.vaos-form__facility-type {
```

Comment:
Would prefer to use a utility class for this, but not sure if there's a way to add a class to a `ui:title`/legend in uiSchema.